src/test/java/net/openhft/chronicle/map/ChronicleMapTest.java

```diff
@@ -1881,6 +1882,52 @@ public void testBytesMarshallableMustBeConcreteValueType() {
         }
     }
 
+    @Ignore //I've been using this class as test harness to probe the map configuration and how it re-sizes.
```

Comment:
I'm adding this test, which doesn't actually test anything, so you can see what the different config items do. There's also some output from the new MapCapacityMonitor here as well.